### PR TITLE
Dependency Rollup (and fixes) for `jetty-9.4.x`

### PR DESCRIPTION
**Pull request by joakime**
Rollup of various dependencies in branch `jetty-9.4.x`

### CODE REVIEW

**Comment by joakime** [comment]
@olamy since you are the release manager right now, i'll let you merge.

But remember, this PR should be a normal merge, **not** a squash+merge.
(A normal merge will close the other 9.4.x open PRs, a squash merge will not)

**Review by olamy — APPROVED** [review]

**Pull request merged**
